ch10.asciidoc

```diff
@@ -116,7 +116,7 @@ Each node verifies every transaction against a long checklist of criteria:
 * The unlocking script (+scriptSig+) can only push numbers on the stack, and the locking script (+scriptPubkey+) must match +IsStandard+ forms (this rejects "nonstandard" transactions).
 * A matching transaction in the pool, or in a block in the main branch, must exist.
 * For each input, if the referenced output exists in any other transaction in the pool, the transaction must be rejected.
-* For each input, look in the main branch and the transaction pool to find the referenced output transaction. If the output transaction is missing for any input, this will be an orphan transaction. Add to the orphan transactions pool, if a matching transaction is not already in the pool.
```

Comment:
I would change it to
`For each input, look in the main branch and the transaction pool to find the transaction referenced by the input.`

and
`If the transaction referenced by the input is missing, this will be an orphan transaction.`

Reply:
@rating89us Thanks for the feedback. I'm going to merge this in. Please feel free to follow-up with further improvements, as you've suggested.

Reply:
Please see #802